### ISSUE TITLE
[BUG] AD resets settings when loading save

### ISSUE THREAD

**Issue by SteelRodent** [Stephan-S/FS19_AutoDrive#1561]
**AutoDrive Version:** 1.1.0.5-RC5
**Release or Custom Zip:** Release  
**Map (If applicable):** All  
**Vehicle / Type (If applicable):**   All driveable
**Multiplayer or Singleplayer:** Singleplayer  

**Describe the bug**  
When changing AD settings they're saved for the current session and according to the config.xml in the savegame folder they're still saved when exiting the game, but as soon as I load the save again, AD returns to default settings.

**If applicable - Steps to reproduce the behavior**  
- Change any setting
- Click apply (or hit enter) and settings are saved.
- Quit game and reload save.

First noticed this behaviour with RC3. AD doesn't throw any errors (nothing to see in log.txt).  Tried disabling CP thinking the interaction on initialization caused it, but it makes no difference - AD still resets on load.

It's immediately noticeable when loading the game because the AD keybinds show up in the "help" menu again, but because I play with corrected fill weights I need to change the stopping distances or AD overshoots when it has to stop or turn.

**Comment by Axel32019** [comment]
Hello, Could you please provide your log file?

**Comment by Axel32019** [comment]
@SteelRodent Could you please test your map Babrosty with the current beta, as I am a little bit busy with other topics:

https://github.com/Stephan-S/FS19_AutoDrive/tree/beta-testing

Code -> Download ZIP
Extract the downloaded zip "FS19_AutoDrive-beta-testing.zip"! Do not put it into the mod folder, this will not work !!!
In the extract you find the folder "FS19_AutoDrive" -> zip and put to mod folder -> start the game

**Comment by kuzkin85** [comment]
machine translation. I have a similar problem. AD started dropping the file to the original settings.
[AutoDrive_No_Man's_Land_config.xml.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/5119809/AutoDrive_No_Man.s_Land_config.xml.txt)
 I should be like this
[AutoDrive_No_Man's_Land_config.xml.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/5119811/AutoDrive_No_Man.s_Land_config.xml.txt)

**Comment by Axel32019** [comment]
Please do not use such special characters, like '
This is causing issues!

**Comment by kuzkin85** [comment]
> Пожалуйста, не используйте такие специальные символы, как '
> Это вызывает проблемы!

Thanks for the help

**Comment by SteelRodent** [comment]
@Axel32019 I got the beta as you requested. In-game version says 1.1.0.5-RC6. It has the same issue with set settings not being loaded, but again, that's without any waypoints set.

I did also try to record a short piece of a route on the same map and saved it, and then AD suddenly loads correctly and apply my custom settings (or at least the config menu says it's using the settings I want). Obviously AD is a bit pointless without any route network, so that's not a big deal, but it shows that it doesn't actually load (or apply) the saved settings if the config doesn't contain any waypoints.
